modules/fbs-core/math-parser/src/main/kotlin/de/thm/ii/fbs/mathParser/transformers/TransformerConfig.kt

```diff
@@ -0,0 +1,14 @@
+package de.thm.ii.fbs.mathParser.transformers
+
+import de.thm.ii.fbs.mathParser.ast.Expr
+import de.thm.ii.fbs.mathParser.ast.Num
+import de.thm.ii.fbs.mathParser.ast.Operator
+import de.thm.ii.fbs.mathParser.ast.UnaryOperation
+import java.math.BigDecimal
+import java.math.RoundingMode
+
+data class TransformerConfig(val decimals: Int, val roundingMode: RoundingMode = RoundingMode.HALF_UP) {
```

Comment:
These lines do not have full code coverage, to get this you have to initialise the class also without the optional parameter and access the variables `decimals` and `roundingMode`. See jacocoReport Report:

![grafik](https://user-images.githubusercontent.com/63304184/220711787-11de31c5-583f-4543-b6b0-b13e98679040.png)